rocketpy/AeroSurface.py

```diff
@@ -202,15 +211,27 @@ def kind(self):
 
     @kind.setter
     def kind(self, value):
-        # Analyze type
+        # Analyzes nosecone type
+        # Sets the k for Cp calculation
+        # Sets the function which creates the respective curve
         self._kind = value
+        value = (value.replace(" ", "")).lower()
+
         if value == "conical":
             self.k = 2 / 3
-        elif value == "ogive":
-            self.k = 0.466
```

Comment:
Hey, this is an important breaking change!

It seems that you are modifying the expected k value of the "ogive" nose kind. Could you elaborate on the reasons for such change please?

Reply:
This was done following the literature found in "THE DESCRIPTIVE GEOMETRY OF NOSECONES" writen by Gary A. Crowell Sr. in 1996.
In the document, several types of nosecone are mentioned alongside their respective equations and k which stand for the coefficient that can be multplied by the length of the nosecone to find it's center of pressure as the following formual implies:

C_p = k * L
Being, 
C_p  ->  Nosecone's center of pressure 
k  ->  coefficient
L  ->  Nosecone's length

The numbers used were extracted straight from the document, which states these values can be calculated through the formula:

X_n  = V / A,  in which
X_n  ->  center of pressure measured from the base to the tip on a nosecone of length = 1m
V  ->  Nosecone's volume
A  ->  Nosecone's base area 

The k coefficient in RocketPy is complementary to X_n on the literature as it's measured from the tip to the base of the nosecone. 

The formulas and defnitions said here can be found on pages 5 and 6 of the beforementioned document and the marked value (k of ogive nosecone = 0.466) is the result of 
k = 1 - X_n 
with the respective C_n for ogive nosecone defined on page 6 of the same document.